### DISCUSSION TITLE
Add support for CoinsEngine

### DISCUSSION THREAD

**Discussion by ZhiBuMiao** [QuickShop-Community/QuickShop-Hikari#1316] [Ideas]
This is a Multi-Currency plugin
Link https://www.spigotmc.org/resources/coinsengine-economy-and-virtual-currencies.84121/

**Comment by Ghost-chu** [comment]
Added via commit https://github.com/Ghost-chu/QuickShop-Hikari/commit/dccfeb6d47849a109b62d2f2508b588edb63eda6
